packages/admin-ui/src/components/05_pages/Content/Content.js

```diff
@@ -169,10 +169,11 @@ class Content extends Component {
       <PageTitle>Content</PageTitle>
       <LoadingBar style={{ position: 'relative', marginBottom: '5px' }} />
       <Paper>
-        {this.props.contentTypes && this.props.actions && (
+        {this.props.contentTypes && this.props.actions.length && (
           <Fragment>
             <div className={styles.filters}>
-              {this.props.contentTypes && this.props.actions && (
+              {/* @TODO Get rid of duplicated condition */}
+              {this.props.contentTypes && this.props.actions.length && (
```

Comment:
Given that this.props.actions is an array, treating it as a boolean doesn't work, as it gets evaluated to true.

Reply:
This could fix one of the source of failures.